app/src/main/java/org/metabrainz/mobile/data/repository/TaggerRepository.kt

```diff
@@ -52,21 +54,39 @@ class TaggerRepository @Inject constructor(private val service: TaggerService) {
         return matchedReleaseData
     }
 
+    fun fetchCoverArt(MBID: String?): LiveData<Resource<CoverArt>> {
```

Comment:
We have an equivalent function in `LookupRepositoryImpl`, let's extract that into a utility function somewhere and reuse that.

Reply:
Done. With this, I have removed the TaggerRepository's separate implementation because LookupRepository should take care of all the lookup calls in the app in one place. Future use cases will require cross using these functions together to give the users a more personalized experience.